src/components/select/multi-select/components.test-pw.tsx

```diff
@@ -284,6 +284,65 @@ export const MultiSelectMultiColumnsComponent = (
   );
 };
 
+export const MultiSelectWithActionButtonComponent = () => {
```

Comment:
suggestion (non-blocking): might be worth just using the list action button to add the new options so it's a bit simpler, opening a dialog seems a bit left field to me

Reply:
Leave this for now until we've had confirmation about how it should behave when the button is clicked